src/main.rs

```diff
@@ -111,9 +213,31 @@ async fn handle_request(
         path_str: path_and_query,
     };
 
+    #[cfg(feature = "expose-metrics")]
+    let start = Instant::now();
+
     let resp = client.raw(raw_request).await.context(RequestIssue)?;
 
+    #[cfg(feature = "expose-metrics")]
+    let end = Instant::now();
+
     debug!("Response: {:?}", resp);
 
+    #[cfg(feature = "expose-metrics")]
+    timing!(METRIC_KEY, start, end, "method"=>m.to_string(), "route"=>p, "status"=>resp.status().to_string());
+
+    info!("{} {}: {}", m, p, resp.status());
```

Comment:
Should this be a trace or something? Seems like it could make the proxy's logs very spammy during use.

Reply:
the idea here is that you can see what the proxy is doing as otherwise it gives you absolutely nothing. this is set at info to be above the existing debug log that logs the raw reply (and is extremely spammy if you just want to take a quick look at what is generally passing by)

Reply:
Disagree a bit here. If I want to see what the proxy is doing, I’d change my environment variable or use the metrics feature. No one running this in production is going to be staring at the logs, they will want Grafana or similar thats easier to keep an eye on.

> the existing debug log...
Just drop that to a trace level. I think the gateway and http crate have raw info at the trace level too.

Reply:
it wasn't ment as a "staring at it in production" thing, more a test phase "are calls even making it to the proxy" thing, without having to log the entire spammy request